$OpenBSD: patch-v8_src_globals_h,v 1.1 2015/10/24 18:22:22 robert Exp $

Try to pre-allocate less memory on OpenBSD in the v8 engine.

--- v8/src/globals.h.orig.port	Tue Oct 13 21:04:45 2015
+++ v8/src/globals.h	Wed Oct 14 07:41:14 2015
@@ -141,7 +141,11 @@ const bool kRequiresCodeRange = true;
 // region. Used only for large object space.
 const size_t kMaximalCodeRangeSize = 256 * MB;
 #else
+#if defined(__OpenBSD__)
+const size_t kMaximalCodeRangeSize = 256 * MB;
+#else
 const size_t kMaximalCodeRangeSize = 512 * MB;
+#endif
 #endif
 #if V8_OS_WIN
 const size_t kMinimumCodeRangeSize = 4 * MB;
